File: Easy/Python/1991. Find the Middle Index in Array.py
class Solution(object):
    def findMiddleIndex(self, nums):
        right = 0
        left = 0

        for i in range(len(nums)):
            right += nums[i]
        for i in range(len(nums)):
            right -= nums[i]
            if left == right:
                return i
            left += nums[i]
        
        return -1

File: Easy/JavaScript/1991. Find the Middle Index in Array.js
var findMiddleIndex = function(nums) {
    let right = 0;
    let left = 0;

    for(let i = 0; i < nums.length; i++)
        right += nums[i];
    for(let i = 0; i < nums.length; i++){
        right -= nums[i];
        if(left == right)
            return i;
        left += nums[i];
    }

    return -1;
};
